storybook/src/Omar/Atoms/Radio/Radio.test.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import "@testing-library/jest-dom/extend-expect";
+import { Radio } from "./Radio";
```

Comment:
Please add another test so you can test if radio is checked or not

Reply:
All done. tests added.

Reply:
Thank you for the tests. We mentioned before to use getByTestId as a last resort. Please consider this as it would make your code better.

Reply:
I used getByLabelTect istead of getByTestId.